CHANGELOG.md

```diff
@@ -7,18 +7,29 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [vNext]
 
-- ...
```

Comment:
Looks like previous versions had it. Don't you want to keep it too? Or maybe remove `vNext` section from the `master` branch completely.

Reply:
I'll keep it. It's useful for adding notes during development.  Won't fix `- ...` this time because it's time-consuming.